nautilusdb/tests/smoke_test.py

```diff
@@ -275,7 +275,7 @@ def test_file_upload_and_ask(self):
             col.ask('question?')
 
             chat1 = col.chat()
-            chat2 = col.chat()
+            chat2 = col.chat(doc_name="text_file.txt")
```

Comment:
verify the returned refs of chat2.ask only refers to text_file.txt.

Reply:
sg, added some asserts